src/test/tests/databases/PLOT3D.py

```diff
@@ -178,5 +179,55 @@ def BasicTests():
 
 BasicTests()
 TestTimeSeries()
-    
+
+# Custom tests with specific views:
```

Comment:
While there isn't really a rule for this, it is helpful to have different sections of a test file in their own functions. This makes it easier in the future to turn off aspects of the test file when trying to debug. (one # comment to turn off a function call rather than having to comment out a bunch of lines of code. )